modules/core/src/main/scala/fs2/kafka/KafkaProducer.scala

```diff
@@ -56,6 +58,53 @@ abstract class KafkaProducer[F[_], K, V] {
 
 object KafkaProducer {
 
+  implicit class ProducerOps[F[_] : MonadThrow, K, V](producer: KafkaProducer[F, K, V]) {
+
+      /**
+        * Produce a single [[ProducerRecord]] without a passthrough value,
+        * see [[KafkaProducer.produce]] for general semantics.
+        */
+      def produceOne_(record: ProducerRecord[K,V]): F[F[RecordMetadata]] = {
+        produceOne(record, ()).map(_.flatMap { res =>
+          val metadata = res.records.head.map(_._2)
+          MonadError[F,Throwable].fromOption(metadata, new IllegalStateException("Exactly one record metadata expected"))
```

Comment:
Is this the only reason we need `MonadThrow`? If so I'd be inclined to just throw an exception here, since this should never happen.